strawberry/fastapi/router.py

```diff
@@ -160,15 +158,9 @@ def __init__(
             GRAPHQL_WS_PROTOCOL,
         ),
         connection_init_wait_timeout: timedelta = timedelta(minutes=1),
-        default: Optional[ASGIApp] = None,
-        on_startup: Optional[Sequence[Callable[[], Any]]] = None,
-        on_shutdown: Optional[Sequence[Callable[[], Any]]] = None,
```

Comment:
do you think it is worth copying the arguments that are available at the moment (plus adding a kwargs for future features)?

I think it would improve the dx a bit 😊

Reply:
I agree. At first I thought to avoid copying, but now I realized that it is better to leave them for additional control and documentation